zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1419,6 +1472,36 @@ components:
         tenantId:
           description: the id of the tenant that owns the job
           type: string
+    JobFailRequest:
+      type: object
+      properties:
+        retries:
+          description: >
+            The amount of retries the job should have left
+          type: integer
+          format: int32
+          default: 0
+        errorMessage:
+          description: >
+            An optional message describing why the job failed. This is particularly useful if a job
+            runs out of retries and an incident is raised, as this message can help explain why an
+            incident was raised.
+          type: string
+          nullable: true
+        retryBackOff:
+          description: >
+            The backoff timeout (in ms) for the next retry.
+          type: integer
+          format: int64
+          default: -1
```

Comment:
Is a backoff of `-1` really appropriate for a user-facing API? A backoff of `0` I can understand as "no backoff, immediately retry". How do I interpret a backoff of `-1`? 🤔

Reply:
I think you are right 😆